CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 - - -
 
+## 0.16.1 (2016-07-15)
+* Remove edx-icons font variable
+
+
```

Comment:
nit: it looks like convention is to end a new section of the changelog with a horizontal rule:

```
---
```